reporting/src/request/utils/chrome-storage-map.js

```diff
@@ -88,6 +88,10 @@ export default class ChromeStorageMap {
     });
   }
 
+  get size() {
+    return this._inMemoryMap.size;
```

Comment:
Careful with this function. It will also count obsolete parameters.

To make it precise, it would need to perform a full scan (O(n) instead of O(1)). But that makes the API easy to misuse, because code like that would be suddenly O(n^2):

for (let i = 0; i < map.size; i += 1) { ... }